### PR TITLE
Various minor fixes

### PR DESCRIPTION
**Pull request by LordAro**
Individual commits probably easier to look at.

Related to #668 (still some work to go though)

Oh, and it's completely untested, so pls try on dev first

### CODE REVIEW

**Comment by AJDurant** [comment]
Looks okay to me, needs testing on -dev though

File: src/Classes/ServiceAPI/ServiceAPI.php [hunk 1]
```diff
@@ -99,43 +99,12 @@ protected function addMixins(&$data, $mixins, $mixin_funcs, $strict = true)
...
-     * a new array.
-     *
-     * @param array $array
-     * @param bool  $full  If true, will return expanded data if available.
-     *
-     * @return array
-     *
-     * @throws MyRadioException Throws an Exception if a provided object is not a DataSource
-     */
-    public static function setToDataSource($array, $full = false)
```
Comment (LloydW93) [inline src/Classes/ServiceAPI/ServiceAPI.php]: This is still used, in, e.g. Controllers\Charts\default/listChartReleases, Controllers\Podcast\default, Controllers\Quotes\default...

File: src/Public/js/myradio.webcam.tracker.js [hunk 2]
```diff
@@ -8,6 +8,10 @@ function webcamTrackViewer() {
     type: "get",
     cache: false,
     url: myradio.makeURL("Webcam", "a-trackViewer"),
+    statusCode: {
+      // API returns 400 when timedelta is too big (minimised tab or whatever) clientside should ignore.
+      400: function () {}
```
Comment (LloydW93) [inline src/Public/js/myradio.webcam.tracker.js]: This is probably good for now, but maybe raise an issue to use the Page Visibility API to stop sending updates?

File: src/Controllers/Scheduler/allocate.php [hunk 3]
```diff
@@ -7,9 +7,9 @@
 
 if ($_SERVER['REQUEST_METHOD'] === 'POST') {
     //Submitted
-    $season = MyRadio_Season::getInstance($_SESSION['myury_working_with_season']);
+    $season = MyRadio_Season::getInstance($_SESSION['myradio_working_with_season']);
```
Comment (LloydW93) [inline src/Controllers/Scheduler/allocate.php]: *is reminded that this thing needs to die in a fire*

**Pull request merged**